main.go

```diff
@@ -26,14 +26,35 @@ var (
 	fileName   string
 	githash    string
 	buildstamp string
+	certFile string
+	keyFile string
 )
 
 const port = 8991
+const httpsPort = 8992
+
+func isProduction() bool {
+	name, err := os.Hostname()
+	if err != nil {
+		panic(err)
+	}
+	if (name == "kommitment.dyn.amicdns.de" || name ==	"Ubuntu-1704-zesty-64-minimal") {
```

Comment:
- Vorschlag: hostnamen in .env auslagern und .env vor Programm-Start source?
- space zwischen == und "Ubuntu"